ipatests/test_integration/test_netgroup.py

```diff
@@ -0,0 +1,183 @@
+#
+# Copyright (C) 2017  FreeIPA Contributors see COPYING for license
+#
+
+import pytest
+
+from ipatests.test_integration.base import IntegrationTest
+from ipatests.test_integration.tasks import clear_sssd_cache
+
+
+def get_test_data():
```

Comment:
Could be test_data a module variable instead of function, I don't see reason to generate the same data again and again